lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +237,28 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        links = article_bs.find_all('a', class_='listing-preview__content')
+        url = ''
+        for link in links:
+            url = link['href']
+        return url
```

Comment:
what is the idea of looping over ALL links, but taking only LAST?

Reply:
done